src/components/config/channels/whatsapp/components/tabs/ProfileTab.vue

```diff
@@ -362,6 +384,43 @@
           });
         }
       },
+      removeEmptyProperties(obj) {
+        for (let key in obj) {
+          if (Object.prototype.hasOwnProperty.call(obj, key)) {
+            let value = obj[key];
+
+            if (typeof value === 'object' && value !== null) {
+              this.removeEmptyProperties(value);
+              if (this.isEmpty(value)) {
+                delete obj[key];
```

Comment:
this can mutate the original object, it's better to return a copy of the object with the attribute filtered out